### ISSUE TITLE
Fields marked "This is never displayed in normal meeting searches."

### ISSUE THREAD

**Issue by gingerwatch** [bmlt-enabled/yap#445]
Currently the "OBSERVER" user can only see information in hidden fields (those marked "This is never displayed in normal meeting searches.") if they are signed in to the ROOT SERVER (like at https://my/root/server/main_server/).

We would like our local OBSERVERS, who can only see meeting information but not make changes, to view hidden information.  The original intent of Observers was so that people manning the Help Line other such trusted servants could have extra info not available to the viewing public.  This is a perfect use of the Observer position, and use of the hidden information.

Currently our Observers do enjoy the different layout and map with pins they get when logging in to our local BMLT server.

But we would love to make use of those hidden fields to include secret info like "Tom R. holds the key to the church, his number is xxx-xxx-xxxx" Or "Lisa F. lives nearby and is willing to give rides, her number is xxx-xxx-xxxx".

**Comment by jbraswell** [comment]
Hi Ginger. What is the actual problem you are trying to solved? Are you requesting a way for volunteers to find contact info for local groups that they’re referring the callers to?

**Comment by radius314** [comment]
Was it intended that this ticket was created on this project?  Or were you referring to the https://github.com/bmlt-enabled/bmlt-root-server?  Or both?

**Comment by jbraswell** [comment]
I asked her on FB to create it here, because the value she's looking for is related to fielding phone calls. I also said that it may get implemented somewhere besides Yap, but that this is the right place to start the discussion, since it's primarily yap users (volunteers) they benefit.

**Comment by jbraswell** [comment]
The original discussion was around volunteers getting access to what's in the hidden fields in the root server so that they can better connect callers with local resources.

My personal opinion is that this shouldn't be implemented in the root server - the root server should just be for CRUD operations and querying via API. I do not know if it should be part of Yap or some new thing.

**Comment by radius314** [comment]
I do see value in it, but the meeting information would not be pulled in Yap ever.  The information is relevant to meetings, information volunteers need to access.  If anything there should be a way to login as an observer and then reload a crouton page and see the information (or other apps).  I feel this is the right approach, because most helplines are not on yap.

**Comment by radius314** [comment]
The cookie from the login, would essentially act as the "token" for the API response to show the additional information.  Assuming they are accessing from the same browser.

**Comment by jbraswell** [comment]
Yeah, I think I agree with you. There could be a new WP plugin basically wraps crouton. It could prompt for a login page, and after you authenticate show meetings with hidden data.

**Comment by radius314** [comment]
This could be part of crouton essentially.  A first pass, I would not build the login page right away.  Crouton would be modified to display the "hidden" fields as part of the data template.  Because you are logged in as an observer, those data points _should_ appear.  If you aren't they are invisible.  If there is enough interest in the feature, build out the login as a convenience feature.  I wouldn't go and build a login until we know if there is traction.

**Comment by jbraswell** [comment]
I like it

**Comment by radius314** [comment]
This is trivial.  I could code this up in a few hours.

**Comment by radius314** [comment]
Moving this ticket to crouton.
